walkthroughs/5-adding-data-sync-graphql/walkthrough.adoc

```diff
@@ -314,7 +314,7 @@ NOTE: the pipe symbol (`|`) allows for multiline input, see link:https://yaml.or
 .. Add the following entry to the *volumeMounts* section:
 +
 ----
-- name: backend-config, mountPath: /tmp/keycloak
+- name: backend-config mountPath: /tmp/keycloak
```

Comment:
we could also use yaml object syntax: `- {name: "backend-config", mountPath: "/tmp/keycloak" }`